book-worm-2010/book-worm-fe
# Feature/header

### What does this PR do? 
>Clearly and concisely describe the feature. 

This feature adds a header component with styling. 
This feature also connects FE&BE through a FETCH request to get all books and be able to search a book as well as POST a book to the student book list.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.
No major issues just small issues from spelling to grabbing the correct nested data. 


### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
src/Card/Card.js
src/Card/_Card.scss 
src/Header/Header.js
src/Header/_Header.scss
src/NewBookForm/NewBookForm.js
src/NewBookForm/_NewBookForm.scss

### How should this be tested?
>What steps need to be taken to confirm functionality?
1. fetch the branch
2. run `npm start` 
3. Click on the Mountain
4. Type in book title and author
5. Depending on your book Type a prediction and click on add book button

The header is at the top styled but has no functionality just yet.

### Steps To Take
>Any questions and possible steps moving forward.
Add functionality to the header as well as change the star to the caterpillar icon.
